cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -466,6 +465,31 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	repoTmpl.Execute(w, data)
 }
 
+func (s *Server) enqueueForIndex(queue *Queue)  func (rw http.ResponseWriter, r *http.Request) {
+	return func(rw http.ResponseWriter, r *http.Request) {
+		if r.Method  != "POST" {
+			http.Error(rw, "not found", http.StatusNotFound)
+			return
+		}
+		err := r.ParseForm()
+		if err != nil {
+			http.Error(rw, "error parsing form", http.StatusBadRequest)
+			return
+		}
+		name := r.Form.Get("repo")
+		if name == "" {
+			http.Error(rw, "missing repo", http.StatusBadRequest)
+			return
+		}
+		opts, err := getIndexOptions(s.Root, name)
+		if err != nil || opts[0].Error != "" {
+			http.Error(rw, "fetching index options", http.StatusInternalServerError)
+			return
+		}
+		queue.AddOrUpdate(name, opts[0].IndexOptions)
```

Comment:
probably want some sort of response like `rw.Write("ok")` :)

Reply:
I assumed a `200 OK` would be enough?